.env.example

```diff
@@ -1,33 +1,6 @@
-CryptoKey="" #generated via a step in the readme
```

Comment:
Split-out the .env file to distinguish between those that are required for docker, vs those that the `server` uses. They used to be all in one .env in the root.